src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3978,13 +4018,17 @@ namespace HeatBalanceSurfaceManager {
 			// Qout,now = (Sum of)(X Tout) - (Sum of)(Y Tin) + (Sum of)(F Qout,old) + (Sum of)(W Qsrc)
 			// In both equations, flux is positive from outside to inside.  The V and W terms are for radiant systems only.
 
-			Real64 const QsrcHist1( QsrcHist( SurfNum, 1 ) );
-
 			// Set current inside flux:
-			Real64 const QH_12 = QH[ l21 ] = TH[ l11 ] * construct.CTFCross( 0 ) - TempSurfIn( SurfNum ) * construct.CTFInside( 0 ) + QsrcHist1 * construct.CTFSourceIn( 0 ) + CTFConstInPart( SurfNum ); // Heat source/sink term for radiant systems
+			Real64 const QH_12 = QH[ l21 ] = TH[ l11 ] * construct.CTFCross( 0 ) - TempSurfIn( SurfNum ) * construct.CTFInside( 0 ) + CTFConstInPart( SurfNum ); // Heat source/sink term for radiant systems
 			if ( surface.Class == SurfaceClass_Floor || surface.Class == SurfaceClass_Wall || surface.Class == SurfaceClass_IntMass || surface.Class == SurfaceClass_Roof || surface.Class == SurfaceClass_Door ) {
-				OpaqSurfInsFaceConduction( SurfNum ) = surface.Area * QH_12;
-				OpaqSurfInsFaceConductionFlux( SurfNum ) = QH_12; //CR 8901
+				if ( construct.SourceSinkPresent ) {
+					Real64 const QH_12s = QH[ l21 ] = QH_12 + QsrcHist( SurfNum, 1 ) * construct.CTFSourceIn( 0 );
```

Comment:
@RKStrand I'm guessing that the small diffs for most files with internal source are coming from here.  Not a problem, really, just a place where the sequence of calculations changed slightly.